spec/unit/actions/synchronize_user_filters_spec.rb

```diff
@@ -143,4 +147,26 @@
       end
     end
   end
+
+  context 'when using sync_multiple_projects_based_on_custom_id mode' do
+    include_context 'using mode with custom_id'
+    let(:params) do
```

Comment:
Similar code found in synchronize_user_filters_spec.rb:94, synchronize_user_filters_spec.rb:153 (mass = 52)